tools/gen_key_materials.sh

```diff
@@ -88,6 +93,17 @@ generate_crypto_materials () {
       -sha256
   fi
 
+  if [ ! -f "${opensk_upgrade}" ]
```

Comment:
Do we want to generate this key pair anyway in case the `force_generate` flag is set?

Reply:
Probably makes sense, yes. The upgrade key should be tied to a AAGUID, and if that is regenerated, then the upgrade key should be as well.